### PR TITLE
feat(*): add v0.8 action data filter's useResults field

### PR DESCRIPTION
**Pull request by lsytj0413**
Signed-off-by: lsytj0413 <511121939@qq.com>

**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:

- add v0.8 actionDataFilter.useResults field

**Special notes for reviewers**:

**Additional information (if needed):**

Refs: https://github.com/serverlessworkflow/sdk-go/issues/70

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
@spolti wanna take a look too?

File: model/workflow.go [hunk 1]
```diff
@@ -206,34 +206,6 @@ func (w *Workflow) setDefaults() {
 	}
 }
 
-// WorkflowRef holds a reference for a workflow definition
-type WorkflowRef struct {
```
Comment (spolti) [inline model/workflow.go:210]: Isn't better keep the Workflow here?

Comment (ricardozanini) [reply model/workflow.go:210]: I think it's fine as it is. It's the same logic as moving `functionRef` to `Action`. This is the only place where `workflowRef` and `functionRef` is used. It's bound to actions.

Comment (lsytj0413) [reply model/workflow.go:210]: This type is only used by Action struct, so i organize them together (in the same file)

**Comment by lsytj0413** [comment]
3ks!

**Comment by spolti** [comment]
I see, thx :)

**Comment by spolti** [comment]
issue: https://github.com/serverlessworkflow/sdk-go/issues/70

**Pull request merged**